<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FSX Fuel Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="HeaderLabel" name="label">
      <property name="text">
       <string>Airplane Information</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="ItemLabel" name="label_2">
        <property name="text">
         <string>Max Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CustomSpinBox" name="spnRange">
        <property name="suffix">
         <string> nmi</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ItemLabel" name="label_3">
        <property name="text">
         <string>Max Fuel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CustomSpinBox" name="spnMaxFuel">
        <property name="suffix">
         <string> lbs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="ItemLabel" name="label_9">
        <property name="text">
         <string>Trip Distance</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CustomSpinBox" name="spnTripDist">
        <property name="suffix">
         <string> nmi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Weight</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="1">
        <widget class="ItemLabel" name="label_5">
         <property name="text">
          <string>Maximum</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CustomSpinBox" name="spnWeightMax">
         <property name="suffix">
          <string> lbs</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CustomSpinBox" name="spnWeightMin">
         <property name="suffix">
          <string> lbs</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="ItemLabel" name="label_4">
         <property name="text">
          <string>Minimum</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnCalc">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HeaderLabel" name="label_6">
      <property name="text">
       <string>Results</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="topMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="ItemLabel" name="label_7">
        <property name="text">
         <string>Trip Fuel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ItemLabel" name="label_8">
        <property name="text">
         <string>Max Cargo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtResultFuel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtResultCargo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>widgets/headerlabel.h</header>
  </customwidget>
  <customwidget>
   <class>ItemLabel</class>
   <extends>QLabel</extends>
   <header>widgets/itemlabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomSpinBox</class>
   <extends>QSpinBox</extends>
   <header>widgets/customspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spnRange</tabstop>
  <tabstop>spnMaxFuel</tabstop>
  <tabstop>spnTripDist</tabstop>
  <tabstop>spnWeightMin</tabstop>
  <tabstop>spnWeightMax</tabstop>
  <tabstop>btnCalc</tabstop>
  <tabstop>txtResultFuel</tabstop>
  <tabstop>txtResultCargo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
